benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1257,5 +1257,6 @@
   {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "citizen", "CRUD": "CRUD"}
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "citizen", "CRUD": "CRUD"},
+  {"LiveFrom": "12/08/2022", "CaseTypeID": "Benefit", "CaseFieldID": "hearingDateIssued", "UserRole": "caseworker-sscs-dwpresponsewriter", "CRUD": "CRUD"}
```

Comment:
Don't add a field to both non-prod and prod sheets, it will cause an error. If you want something behind a feature flag and not to go to production, add it to a nonprod sheet.